packages/tenant-process/src/services/tenantService.ts

```diff
@@ -1708,9 +1708,6 @@ function reassignVerifiedAttribute(
             organizationId,
             tenantAttributeSeed.expirationDate
           ),
-          revokedBy: verifiedTenantAttribute.revokedBy.filter(
-            (i) => i.id !== attr.id
-          ),
```

Comment:
I am not sure about this.
@beetlecrunch should the entry be removed when a verifier assigns a verified attribute again, after having already revoked it previously?

Reply:
In the Scala counterpart there is no mention of removing the entry from the `revokedBy`, maybe is happens in the management?

Reply:
you are right, in Scala it is not removed 🤔 
But IMO that is a bug, because the assumption was to keep the attribute either in `verifiedBy` or in `revokedBy`, but not in both.
In case I'm missing something, let's wait for @beetlecrunch

Reply:
The agreement service checks if the attribute is verified through the `verifiedBy` field even if a `revokedBy` exists for the same attribute.
I suppose this behavior was implemented to have a sort of history of the verification/revocation, but maybe it's simply a bug.
Let's remove the entry in the `revokedBy` when an attribute is reassigned.

Reply:
Great! Thanks for the clarification, gonna add it back.
This PR is still needed since the filter logic was wrong

Reply:
This change also need an intervention in the frontend